rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/service_introspection.hpp

```diff
@@ -26,11 +26,18 @@
 namespace rosidl_typesupport_introspection_cpp
 {
 
+/// This struct provides introspection information for one service definition.
+/// A service is comprised of two interfaces: the request and the response.
 typedef struct ServiceMembers_s
 {
+  /// The namespace in which the service resides, e.g. "example_messages__srv" for
```

Comment:
Since this is the C++ API, I think this string will be `example_messages::srv` instead.

Reply:
+1, changed f6ff3f6